package.json

```diff
@@ -38,10 +38,7 @@
     "react": "global:React",
     "react-dom": "global:ReactDOM"
   },
-  "peerDependencies": {
-    "react": ">=0.14",
-    "react-dom": ">=0.14"
-  },
+  "peerDependencies": {},
```

Comment:
I believe these were present in `devDependendencies` in the `dev` branch, and needed to be removed from there before merging, but we should keep them in `peerDependencies`.